apps/api/src/endpoints/view/view.controller.ts

```diff
@@ -28,24 +29,24 @@ export class ViewController {
   }
 
   @Get('/proposals/:nonce')
-  @ApiQuery({ name: 'nonce', description: 'The proposal\'s nonce', required: false })
+  @ApiParam({ name: 'nonce', description: 'The proposal\'s nonce', required: false })
   async getGovernanceProposal(
     @Param('nonce', ParseIntPipe) nonce: number,
   ): Promise<GovernanceProposal> {
     return await this.viewService.getProposalDetails(nonce);
   }
 
   @Get('/voting-power/:address')
-  @ApiQuery({ name: 'address', description: 'The address to fetch the voting power for', required: false })
+  @ApiParam({ name: 'address', description: 'The address to fetch the voting power for', required: false })
   async getVotingPower(
     @Param('address', ParseAddressPipe) address: string,
   ): Promise<GovernanceVotingPower> {
     return await this.viewService.getAddressVotingPower(address);
```

Comment:
add also `!account` check -> to throw `NotFoundException`

Reply:
The address is validated by ParseAddressPipe and if the address does not have any voting power, the endpoint will return `0`